201-web-app-github-deploy/azuredeploy.json

```diff
@@ -51,7 +51,7 @@
         },
         "repoURL": {
             "type": "string",
-            "defaultValue": "https://github.com/Microsoft/project-html-website",
+            "defaultValue": "https://github.com/leestott/WebSiteTest.git",
```

Comment:
We can't use a personal repo here - is there nothing supported in github we can use?

Reply:
@bmoore-msft  what about https://github.com/Azure-Samples/app-service-web-html-get-started.git is this good?

Reply:
Changed the repo.

Reply:
@bmoore-msft can you please check this over and merge if your happy.